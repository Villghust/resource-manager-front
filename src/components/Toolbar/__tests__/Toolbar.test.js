import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import { Toolbar } from '../index';

it('renders correctly', () => {
    const tree = renderer
        .create(
            <BrowserRouter>
                <Toolbar />
            </BrowserRouter>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
