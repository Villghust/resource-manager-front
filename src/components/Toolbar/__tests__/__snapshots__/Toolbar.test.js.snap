// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-3 MuiToolbar-gutters"
    >
      <a
        className="makeStyles-root-4"
        href="/"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root makeStyles-link-2 MuiTypography-h6"
        >
          Agenda
        </h6>
      </a>
      <a
        className="makeStyles-root-4"
        href="/users"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root makeStyles-link-2 MuiTypography-h6"
        >
          Colaboradores
        </h6>
      </a>
      <a
        className="makeStyles-root-4"
        href="/resources"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root makeStyles-link-2 MuiTypography-h6"
        >
          Recursos
        </h6>
      </a>
      <a
        className="makeStyles-root-4"
        href="/reports"
        onClick={[Function]}
      >
        <h6
          className="MuiTypography-root makeStyles-link-2 MuiTypography-h6"
        >
          Relat√≥rios
        </h6>
      </a>
    </div>
  </header>
</div>
`;
